web/src/components/FileBrowser/FileUploadInstructions.vue

```diff
@@ -53,4 +53,12 @@ import { useDandisetStore } from '@/stores/dandiset';
 
 const store = useDandisetStore();
 const dandisetIdentifier = computed(() => store.dandiset?.dandiset.identifier);
+
+if (dandisetIdentifier.value === undefined) {
+  throw new Error('store.dandiset must be defined');
+}
+
+const downloadCommand = computed(() => {
+  return `dandi download ${window.location.origin}/dandiset/${dandisetIdentifier.value}/draft`
```

Comment:
I did not really follow the discussion in the [other thread](https://github.com/dandi/dandi-archive/pull/1810/files#r1466534015). Could you summarize here whether this logic is correct for the upload instructions? Is there a reason not to use the same logic as in the download dialog below? Specifically, if a user downloads their dandiset via `dandi download DANDI:123456/draft`, will the subsequent `dandi upload` command get confused about where to upload?

Reply:
The DANDI CLI _should_ still be OK -- as @yarikoptic mentioned that any short-hand prefix shouldn't break here, as "The dandi-cli is actually "too smart" to support a wide range of URLs and they do not have to be associated with a specific "short named" instance (if something fails -- it is a bug)"

I can test further if you'd like more confirmation